.gitignore

```
@@ -15,3 +15,4 @@ vendor/
 
 *.kubeconfig
 cover.out
```

This will also need to be updated 🙂 
```suggestion
coverage.out
```